pkg/fuzz/analyzers/time/analyzer.go

```diff
@@ -55,28 +56,45 @@ func (a *Analyzer) ApplyInitialTransformation(data string, params map[string]int
 				gologger.Warning().Msgf("Invalid sleep_duration parameter type, using default value: %d", duration)
 			}
 		}
+		// Default unit is second. If we get passed milliseconds, multiply
+		if unit, ok := params["time_unit"]; ok {
+			duration = a.handleCustomTimeUnit(unit.(string), duration)
+		}
```

Comment:
Wouldn't it be more reliable to use `time.ParseDuration` directly and skip having a custom handler for it? So we could remove the "time_unit" param entirely and make "sleep_duration" a string that gets parsed into `time.Duration`. What do you think, @Ice3man543?

Reply:
https://github.com/projectdiscovery/nuclei/pull/5781/commits/311198f7e76a4a70b2f152528daafe99eddb67a1#diff-74de151d167a5a2baddd58e1147646f15613a65f9e19645a8a54e8895a55f096R947 - done here